email_hide_footer/models/ir_ui_view.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2025 Quartile (https://www.quartile.co)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+import re
+
+from odoo import models
+
+
+class IrUIView(models.Model):
+    _inherit = "ir.ui.view"
+
+    def _render(self, values=None, engine="ir.qweb", minimal_qcontext=False):
+        rendered_content = super()._render(values, engine, minimal_qcontext)
+        if self.env.context.get("hide_footer"):
+            footer_pattern = re.compile(
+                r'<p\s*style\s*=\s*["\']color:\s*#555555;\s*margin-top:\s*32px;["\']>'
+                r"\s*Sent.*?</p>",
```

Comment:
This looks a bit too specific and fragile. I think it may be safer to go with template inheritance.